### DISCUSSION TITLE
[资料]新艾利都教育考试院发布的2024年中考录取分数线

### DISCUSSION THREAD

**Discussion by RWHwaxili** [share121/inter-knot#5304] [其他]
2024新艾利都中考文化分680 体育60 总分740
### 第一批次控制分数线 598分
艾利都一中雅努斯校区   673分
艾利都一中二分街校区   675分
艾利都二中五分街校区   702分
艾利都二中外环校区   704分
艾利都三中   687分
艾利都七中   666分
艾利都十中七分街校区   652分
艾利都十九中   677分
艾利都师大附中   712分
艾利都财大附中   699分
艾利都实验中学   637分
艾利都铁路一中   645分
艾利都外国语学校   640分
艾利都五中（民办）   631分
艾利都当代学校（民办）   598分
### 第二批次控制分数线 485分
艾利都六中   605分
艾利都八中   601分
艾利都十六中三分街校区   599分
艾利都十六中雅努斯校区   595分
艾利都朝阳中学   581分
艾利都十五中   581分
艾利都十二中   555分
艾利都十三中   543分
艾利都十八中   520分
艾利都二十三中   533分
艾利都二十六中   596分
艾利都铁路二中外环校区   608分
艾利都新世纪学校（民办）   485分
艾利都HIA学校C41空洞校区（民办）   485分
艾利都HIA学校零号空洞校区（民办）   485分
### 中职学校/技工学校指导分数线 350分
（相关资料丢失）

**Comment by yansai0v0** [comment]
看来还得努力，争取明年拿个好成绩😟

**Comment by 114514qwe-der** [comment]
不是吧！？

**Comment by fzdqz** [comment]
艾利都外国语学校存在的意义是什么🤔

**Reply by RWHwaxili** [reply]
没准学的是邦布语
（嗯呢，嗯呐哒，嗯呢呢）

**Comment by AAAAAnaye** [comment]
许愿一中上岸！！

**Comment by PekingZ** [comment]
兄弟们！铁路一中咋样啊？报的这个学校，不会以后要去工地打灰吧😰

**Reply by RWHwaxili** [reply]
属于一类重点，升本率不错
看帖子 #5393

**Comment by fengxuejia** [comment]
要读初四了😭
